upup/models/cloudup/resources/addons/aws-cloud-controller.addons.k8s.io/v1.18.0.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: aws-cloud-controller-manager
+  namespace: kube-system
+  labels:
+    k8s-app: aws-cloud-controller-manager
+spec:
+  selector:
+    matchLabels:
+      k8s-app: aws-cloud-controller-manager
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        k8s-app: aws-cloud-controller-manager
+    spec:
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      tolerations:
+      - key: node.cloudprovider.kubernetes.io/uninitialized
+        value: "true"
+        effect: NoSchedule
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
+      serviceAccountName: cloud-controller-manager
+      containers:
+        - name: aws-cloud-controller-manager
+          image: gcr.io/k8s-staging-provider-aws/cloud-controller-manager:v1.18.0-alpha.1
```

Comment:
Does the version need to track that of Kubernetes? The cloud-provider-aws repo doesn't say what the version compatibility policy is.

Reply:
You can most likely just copy the OpenStackCCMTag function in template_functions.go.